nautobot/core/apps/__init__.py

```diff
@@ -237,12 +439,12 @@ def __init__(self, link, name, permissions=None, buttons=None, weight=1000):
         self.link = link
         self.name = name
         self.weight = weight
-
-        if buttons is not None and not isinstance(buttons, (list, tuple)):
-            raise TypeError("Buttons must be passed as a tuple or list.")
-        elif not all(isinstance(button, NavMenuButton) for button in buttons):
-            raise TypeError("All buttons defined in an item must be an instance or subclass of NavMenuButton")
-        self.buttons = buttons
+        if buttons is not None:
+            if not isinstance(buttons, (list, tuple)):
+                raise TypeError("Buttons must be passed as a tuple or list.")
+            elif not all(isinstance(button, NavMenuButton) for button in buttons):
+                raise TypeError("All buttons defined in an item must be an instance or subclass of NavMenuButton")
+            self.buttons = buttons
```

Comment:
As written here, if `buttons is None` then `self.buttons` won't get initialized - seems wrong to me.